pkg/go/transformer/dsltojson.go

```diff
@@ -103,7 +103,6 @@ func (l *OpenFgaDslListener) ExitRelationDeclaration(ctx *parser.RelationDeclara
 					},
 				},
 			}
-			break
```

Comment:
why remove break?  You mean it to be fall through switch/case logic?

Reply:
Unlike JS and Java, Go does not need `break` between cases - you only use it if you need to break early, it does not have auto fallback to other cases.